images/go-grpc-greeter-server/rootfs/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:buster as build
+
+WORKDIR /go/src
```

Comment:
I think it would be better `/go/src/greeter-server`

Reply:
changed